javascript/javascript1/week4/sentimentAnalyzer.js

```diff
@@ -0,0 +1,36 @@
+const allowedPositiveWords=["super", "awesome", "happy", "excited"];
+const allowedNegativeWords=["hate", "boring", "sad", "unhappy"];
+
+function getSentimentScore(incomingString)
+{
+    let result=
+    {
+        scrore:0,
+        positiveWords:[],
+        negativeWords:[]
+    }
+    const wordString=incomingString.split('');
```

Comment:
maybe it's better to call this `words`, because it's an array of strings, not a string.